server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java

```diff
@@ -739,6 +748,16 @@ private RestResponse getDetailResponse(RestRequest request, Cache<?, ?> cache, b
       return addEntityAsJson(fullDetail.toJson(), invocationHelper.newResponse(request), pretty).build();
    }
 
+   private SearchStatistics getSearchStatistics(Cache<?, ?> cache) {
```

Comment:
As we only use `SearchStatistics` to set `indexingInprogres`, I think it would be cleaner just to create a method like `boolean isIndexingInProgress(Cache<?,?>)`. That way all logic is encapsulated in a single method.

Reply:
good idea